concourse/pipelines/rhui-release.yaml

```diff
@@ -0,0 +1,175 @@
+jobs:
+- name: manual-trigger
+  plan:
+  - get: cds-image
+    trigger: false
+  - get: rhua-image
+    trigger: false
+- name: deploy-staging-us-west1
+  plan:
+  - get: cds-image
+    passed: [ ]
+    trigger: true
+  - get: rhua-image
+    passed: [ ]
+    trigger: true
+  - config:
+      image_resource:
+        source:
+          repository: google/cloud-sdk
+          tag: alpine
+        type: registry-image
+      platform: linux
+      run:
+        args:
+        - -exc
+        - gcloud compute instance-groups managed rolling-action replace --quiet rhua-mig-staging-us-west1 --region=us-west1 --project=google.com:rhel-infra; gcloud compute instance-groups managed wait-until --version-target-reached --quiet rhua-mig-staging-us-west1 --region=us-west1 --project=google.com:rhel-infra;
+        path: sh
+    task: deploy-rhua
+  - config:
+      image_resource:
+        source:
+          repository: google/cloud-sdk
+          tag: alpine
+        type: registry-image
+      platform: linux
+      run:
+        args:
+        - -exc
+        - gcloud compute instance-groups managed rolling-action replace --quiet cds-mig-staging-us-west1 --region=us-west1 --project=google.com:rhel-infra; gcloud compute instance-groups managed wait-until --version-target-reached --quiet cds-mig-staging-us-west1 --region=us-west1 --project=google.com:rhel-infra;
+        path: sh
+    task: deploy-cds
+- name: wave-0-finished
+  plan:
+  - get: cds-image
+    passed:
+    - deploy-staging-us-west1
+    trigger: true
+  - get: rhua-image
+    passed:
+    - deploy-staging-us-west1
+    trigger: true
+- name: deploy-prod-europe-west1
+  plan:
+  - get: cds-image
+    passed:
+    - wave-0-finished
```

Comment:
since these are going to look like 'triggers' to each subsequent wave, do you think it makes sense to name them after the wave they're triggering, rather than what they're 'finished' by? e.g. name this one 'wave-1'